winui/DataGrid/Row-Drag-and-Drop.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: post
+title: Row drag and drop in WinUI DataGrid control | Syncfusion
+description: Learn here all about row drag and drop support in Syncfusion WinUI DataGrid (SfDataGrid) control and more.
+platform: winui
+control: SfDataGrid
+documentation: ug
+---
+
+# Row drag and drop in WinUI DataGrid (SfDataGrid)
+
+[WinUI DataGrid](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.SfDataGrid.html) allows drag and drop the rows by setting the `AllowDraggingRows` and [AllowDrop](https://docs.microsoft.com/en-us/dotnet/api/system.windows.uielement.allowdrop) property as `true`. You can also drag and drop the rows between datagrid and other controls such as `ListView` , `SfTreeView` and `SfTreeGrid`. 
```

Comment:
[WinUI DataGrid](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.SfDataGrid.html) allows you to drag and drop the rows by setting the `AllowDraggingRows` and [AllowDrop](https://docs.microsoft.com/en-us/dotnet/api/system.windows.uielement.allowdrop) properties to `true`. You can also drag and drop the rows between DataGrid and other controls such as `ListView`, `SfTreeView`, and `SfTreeGrid`.